jaeseon/db/migrations/20221107110916_create_likes_table.sql

```diff
@@ -0,0 +1,11 @@
+-- migrate:up
+CREATE TABLE likes (
+    id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+    user_id INT NOT NULL,
+    post_id INT NOT NULL,
+    CONSTRAINT likes_user_id_fkey FOREIGN KEY (user_id) REFERENCES users(id),
+    CONSTRAINT likes_post_id_fkey FOREIGN KEY (post_id) REFERENCES posts(id)
+);
```

Comment:
```suggestion
    CONSTRAINT likes_user_id_fkey FOREIGN KEY (user_id) REFERENCES users(id),
    CONSTRAINT likes_post_id_fkey FOREIGN KEY (post_id) REFERENCES posts(id)
    CONSTRAINT likes_user_id_post_id_unq UNIQUE (user_id, post_id)
);
```
likes 테이블에도 추가로 제약 조건이 필요합니다. 한사람은 똑같은 게시글에 대해서 중복으로 좋아요를 누를 수 없습니다. 같은 사람이 동일한 게시글에 한 번 더 좋아요 request가 들어올 때 data row가 중복으로 생기는 것을 방지하기 위해 UNIQUE 제약을 다음과 같이 복수의 column에 묶어서 걸어줄 수 있습니다! 수정해주세요~